webpack.config.js

```diff
@@ -59,6 +59,6 @@ module.exports = {
         concurrency: 100,
       },
     }),
-    new webpack.DefinePlugin(helpers.envKeys),
+    //new webpack.DefinePlugin(helpers.envKeys),
   ],
```

Comment:
You can just remove this instead of commenting.

Reply:
Ok

Reply:
Done